Detections/PulseConnectSecure/MultipleFailedVPNUserLogins.yaml

```diff
@@ -0,0 +1,26 @@
+id: 34663177-8abf-4db1-b0a4-5683ab273f44
+name: Multiple Failed VPN User Logins
+description: |
+  'Creates an incident in the event of multiple unsuccessful attempts to log into the VPN server, which could indicate a potential brute force attack.'
+severity: Low
+requiredDataConnectors:
+  - connectorId: PulseConnectSecure
+    dataTypes: 
+      - Syslog
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - CredentialAccess 
+relevantTechniques:
+  - T1110
+query: |
+
+  let timeframe = ago(1h);
+  PulseConnectSecure
+  | where TimeGenerated >= timeframe
+  | where Messages contains "Login failed"
+  | summarize count() by User, Source_IP
```

Comment:
Looks like you missed the summarize here -  | summarize StartTime = min(TimeGenerated), EndTime = max(TimeGenerated), count() by User, Source_IP

Reply:
apologies, corrections made.